### PR TITLE
{tools}[intel/2020a] snakemake v5.26.1 (+ GitPython v3.1.9) w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/snakemake/snakemake-5.26.1-intel-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        'checksums': ['9d8ecf532868cc31fa9ff59ee5d5b2049600c5c902c18c794a2bad677c1f92e5'],
+    }),
+    (name, version, {
+        'checksums': ['94cfed2f47f6441cca6cf9d53cffdc11eb78dd1758d96c0af9cd5e453247363e'],
+    }),
+]
+
+# site-specific: the line below is the command for loading this module with the (default) EB module naming scheme
+# if necessary, change this command to match your cluster setup
+local_snakemake_load = 'module load %(name)s/%(version)s-%(toolchain_name)s-%(toolchain_version)s%(versionsuffix)s'
```
Comment (boegel) [inline easybuild/easyconfigs/s/snakemake/snakemake-5.26.1-intel-2020a-Python-3.8.2.eb:73]: @Darkless012 Not a blocker for this PR, but we should come up with a better approach for this, for example by adding support to EasyBuild framework for a new `%(mod_name)s` template value, so the active module naming scheme can be taken into account without having to tweak the easyconfig file.

Please open an issue on this in framework, and maybe try to take a look at implementing it?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3573.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/41a6668c736fc53c99ccac1b80bed2ea for a full test report.

File: easybuild/easyconfigs/s/snakemake/snakemake-5.26.1-intel-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/snakemake/snakemake-5.26.1-intel-2020a-Python-3.8.2.eb:73]: Quickly tackled that myself, see https://github.com/easybuilders/easybuild-framework/pull/3497

Comment (Darkless012) [reply easybuild/easyconfigs/s/snakemake/snakemake-5.26.1-intel-2020a-Python-3.8.2.eb:73]: Will take a look :) Thanks

Comment (verdurin) [reply easybuild/easyconfigs/s/snakemake/snakemake-5.26.1-intel-2020a-Python-3.8.2.eb:73]: @boegel do you want to wait for this to incorporate the framework improvement?

Comment (boegel) [reply easybuild/easyconfigs/s/snakemake/snakemake-5.26.1-intel-2020a-Python-3.8.2.eb:73]: Yeah, https://github.com/easybuilders/easybuild-framework/pull/3497 is merged now, so we should leverage the new `%(module_name)s` template here.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/4abcd2e5ae33a6e6760cdc79c53b85f5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3570.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/d408391330b2485c5454140c0dbe7910 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/ca534368756508c8b8ae54d88ce469be for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**